source/localizable/tutorial/subroutes.md

```diff
@@ -0,0 +1,361 @@
+
+  const model = {
+    title: "Grand Old Mansion",
+    owner: "Veruca Salt",
+    city: "San Francisco"
+  };
+
+  const controller = {};
+
+  route.setupController(controller, model);
+
+  const expectedController = {
+    rental: model
+  };
+
+  assert.deepEqual(controller, expectedController, 'rental is added');
+});
+
+```
+
+Here, we ensure that if our model returns an object, that it is renamed to `rental` on our controller.
+
+# Adding the Rental To Our Template
+
+Next, we can update the template for our show route (`app/templates/rentals/show.hbs`) and list the information for our rental.
+
+```app/templates/rentals/show.hbs
+{{rental-listing rental=rental}}
+
+<article class="listing">
+  <h3>More about {{rental.title}}</h3>
+  <div class="detail">
+    {{rental.description}}
+  </div>
+</article>
+```
+
+Notice that we can access our `model` as `rental` because of the way we used `setupController`.
+We pass our `rental` to the `rental-listing` component and add a new section listing the rental's `title` and `description`.
+Browse to `localhost:4200/rentals/grand-old-mansion` and you should see the information listed for that specific rental.
+
+# Adding an Index Sub-route
+
+One of the first things you may notice is that if you browse directly to `localhost:4200/rentals`,
+all we see are the items from our parent route (`app/template/rentals.hbs`).
+This is because we do not have an `index` sub-route.
+Much like how our application defaults to the `index` route,
+our `rentals` route defaults to `index` and tries to render it in our `outlet`.
+
+We will want to generate an `index` route and use it to list all of our rentals and allow the user to select one to view additional information.
+
+```shell
+ember g route rentals/index
+```
+
+If you open up your Router (`app/router`) you may notice that nothing has updated.
+
+```app/router.js
+Router.map(function() {
+  this.route('about');
+  this.route('contact');
+  this.route('rentals', function() {
+    this.route('show', { path: '/:slug' });
+  });
+});
+```
+
+Much like how our applications `index` route doesn't appear in our Router, `index` routes on sub-routes won't explicitly appear in the Router either.
+Ember knows that the default action is to take the user to the `index` route.
+However, you can modify the `index` route if needed.
+For example, you can modify the `path` much like we do with our `show` route by adding `this.route('index', { path: '/custom-path'})`.
+
+Next, let's modify the our `index` route.
+
+```app/routes/rentals/index.js
+export default Ember.Route.extend({
+  model() {
+    return this.store.findAll('rental');
+  }
+});
+```
+
+This will return all of our rentals.
+Let's list all of our rentals on our template (`app/templates/rentals/index.hbs`).
+
+```app/templates/rentals/index.hbs
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{rental-listing rental=rentalUnit}}
+    {{link-to "Select Rental" "rentals.show" rentalUnit.slug}}
```

Comment:
Can we surround {{rental-listing}} and make that area clickable.  I'd rather just be able to click on the rental listing to show details.  Maybe we can update the css plugin to make it even darker when mousing over to make it obvious that its clickable.